app/controllers/replies_controller.rb

```diff
@@ -294,8 +259,13 @@ def add_to_multi_reply(reply, reply_params)
 
     # Set up editor
     @post = reply.post
-    empty_reply_hash = permitted_params.permit(:character_id, :icon_id, :character_alias_id)
+    empty_reply_hash = permitted_params.permit(:character_id, :character_alias_id)
     @reply = @post.build_new_reply_for(current_user, empty_reply_hash)
+    if @reply.character_id.nil?
+      @reply.icon_id = current_user.avatar_id
+    else
+      @reply.icon_id = @reply.character.default_icon.try(:id)
+    end
```

Comment:
```suggestion
    @reply.assign_default_icon
```
and extract this here in and `build_new_reply_for`?